definitions/npm/base64-arraybuffer_v1.x.x/flow_v0.83.x-/base64-arraybuffer_v1.x.x.js

```diff
@@ -0,0 +1,5 @@
+declare module 'base64-arraybuffer' {
+  declare function encode(arraybuffer: ArrayBuffer): string;
+
+  declare function decode(base64: string): ArrayBuffer;
```

Comment:
I don't think these should be usable unless you decided to export them
```suggestion
  declare export function encode(arraybuffer: ArrayBuffer): string;

  declare export function decode(base64: string): ArrayBuffer;
```